tests/functional/functional_test.py

```diff
@@ -65,13 +67,15 @@ def setUp(self):
         self.journalist_location = "http://localhost:%d" % journalist_port
 
         def start_source_server():
+            Random.atfork()
```

Comment:
Can you add a short comment with maybe a link to some external documentation why this Random.atfork() was necessary?